apps/web/src/app/(main)/(settings)/settings/(account)/profile/page.tsx

```diff
@@ -16,8 +16,8 @@ export default function SettingsProfilePage() {
             Deleting an account is irreversable and cannot be undone! Once you
             delete your account—your characters, images, and account history
             will be completely wiped from our servers to comply with GDPR
-            standards. If you are certain that you'll delete your account, it's
-            important to export your data first before proceeding.
+            standards. If you are certain that you{"'"}ll delete your account,
+            it{"'"}s important to export your data first before proceeding.
```

Comment:
This looks awkward - this is probably something to do with the ESLint rule of React's unescape characters or something; it's nothing major to revert this as it's more of a cosmetic change, and there are no viz changes when viewed through browsers, I'd suggest reverting it and disabling the said rule

```suggestion
            standards. If you are certain that you'll delete your account, it's
            important to export your data first before proceeding.
```